repytah/search.py

```diff
@@ -347,7 +347,8 @@ def find_all_repeats(thresh_mat, bw_vec):
     
     Returns
     -------
-        
+        all_lst np.array
```

Comment:
I already added this actually - not sure if it's going to cause a merge conflict...